recipes/pyretis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyretis" %}
+{% set version = "0.9.2.dev2" %}
```

Comment:
If there is not a non-`dev` release, that's fine. We try to distinguish `dev` releases (by releasing under a different label) if possible to avoid disruption for end users. This only really matters if there are non-`dev` releases that we can package. However it sounds like there are only `dev` releases (this is confirmed by PyPI). So there is nothing more to do here.

TL;DR Don't worry about it. This is fine.